internal/cli/apis.go

```diff
@@ -156,6 +157,7 @@ func listApisCmd(cli *cli) *cobra.Command {
 	}
 
 	cmd.Flags().BoolVar(&cli.json, "json", false, "Output in json format.")
+	cmd.Flags().BoolVar(&cli.csv, "csv", false, "Output in csv format.")
```

Comment:
Still think it would be wise to validate to make the usage mutually exclusive. Fortunately, Cobra allows us to express this with the `MarkFlagsMutuallyExclusive` function. There is precedent in this codebase: https://github.com/search?q=repo%3Aauth0%2Fauth0-cli%20mutually%20exclusive&type=code

```suggestion
	cmd.Flags().BoolVar(&cli.json, "json", false, "Output in json format.")
	cmd.Flags().BoolVar(&cli.csv, "csv", false, "Output in csv format.")
	cmd.MarkFlagsMutuallyExclusive("json", "csv")
```

Granted, it is an extraneous line of code for each command, but I think it improves the DX.

Reply:
Agreed, otherwise this PR is solid 💯 , nice one @m3talsmith